### DISCUSSION TITLE
좋댓구알 스터디 for Algorithm

### DISCUSSION THREAD

**Discussion by s9hn** [like-comment-subscribe-and-Algorithm/sehunTV#1] [Announcements]
## 스터디 시작 전 꼭 읽어봤으면 하는 글

[프로그래밍과 알고리즘 공부 방법 - 김창준님](https://seamless.tistory.com/96#google_vignette)

알고리즘 부분만 읽어보셔도 좋습니다.

## 진행 방식(new!)

[빅오 표기법(Big-O notation), 시간복잡도, 공간복잡도](https://blog.naver.com/PostView.naver?blogId=kks227&logNo=220769859177&parentCategoryNo=&categoryNo=299&viewDate=&isShowPopularPosts=false&from=postList)

## **주 1회 스터디 회의(23시)**

- 새로운 챕터(1개) 이론 훑기
    - 이론은 돌아가면서 진행
- 공통 문제 풀어보기 및 공유

**스터디 1 사이클 당 5문제(평일 하루 1문제, 분류 자유)**

[문제은행](https://www.acmicpc.net/step)

**페널티**

- 매주 회의 때 마다, 공동계좌에 (못 푼 문제 * 1000원) 기부 ^^

**PR컨벤션**

- 주차별로 공통과제만 PR작성
- 개인과제는 자유
- 별도의 컨벤션 없음

**Comment by KWY0218** [comment]
@s9hn 
혹시 다음주 월욜날 풀 공통 문제는 정해졌었나요??
